build/cloud/03-clusterrole.yaml

```diff
@@ -7,14 +7,17 @@ metadata:
     kubeedge: cloudcore
 rules:
 - apiGroups: [""]
-  resources: ["nodes", "nodes/status", "configmaps", "pods", "pods/status", "secrets", "endpoints", "services","persistentvolumes","persistentvolumeclaims"]
+  resources: ["serviceaccounts/token", "nodes", "nodes/status", "configmaps", "pods", "pods/status", "secrets", "endpoints", "services", "persistentvolumes", "persistentvolumeclaims"]
   verbs: ["get", "list", "watch", "create", "update"]
 - apiGroups: [""]
   resources: ["namespaces"]
   verbs: ["get", "create"]
 - apiGroups: [""]
   resources: ["pods/status"]
   verbs: ["patch"]
+- apiGroups: [""]
+  resources: ["nodes"]
```

Comment:
Maybe we just need node/status here?

Reply:
Look at the error log here:
https://github.com/kubeedge/kubeedge/pull/3449#issuecomment-989957547

Reply:
Shall we combine two sections together?
- apiGroups: [""]
  resources: ["nodes","pods/status"]
  verbs: ["patch"]

Reply:
> Shall we combine two sections together?
> 
> * apiGroups: [""]
>   resources: ["nodes","pods/status"]
>   verbs: ["patch"]

Done.